spec/models/state_file_base_intake_spec.rb

```diff
@@ -442,4 +441,81 @@
       end
     end
   end
+
+  describe "#other_intake_with_same_ssn_has_submission?" do
+    let(:intake) { create :state_file_nc_intake, hashed_ssn: hashed_ssn }
```

Comment:
this is a method on every state, but only testing with nc intake, is that ok? Also noting that the specs were added to `state_file_az_intake` only for the `should_be_sent_reminder` methods, but maybe it should be moved into the base intake since this method should apply to every state?

Reply:
Agreed, I think - behavior in `StateFileBaseIntake` should be tested in `state_file_base_intake_spec.rb`

Reply:
And to your other question - as long as it's not overridden on any of the subclasses, I think it's fine to only test it on an instance of one of the subclasses.